charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -56,16 +54,13 @@ alfresco-content-services:
 #    image:
 #      repository: quay.io/alfresco/alfresco-dbp-share
 #      tag: "0.3-SNAPSHOT"
+  
 alfresco-sync-service:
   enabled: false
   alfresco-infrastructure:
     enabled: false
 
 alfresco-infrastructure:
   enabled: true
-  #DEPLOY-613 - Multiple ActiveMQ instances are deployed
-  #TODO Remove as part of DEPLOY-563
   activemq:
-    enabled: false  
-
-  
+    enabled: true
```

Comment:
This was added as a workaround. We should remove it rather than set it to true.

Reply:
Done. Thank you! 👍